specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-07-01/managedClusters.json

```
@@ -4183,6 +4186,28 @@
       "title": "The max price (in US Dollars) you are willing to pay for spot instances. Possible values are any decimal value greater than zero or -1 which indicates default price to be up-to on-demand.",
       "description": "Possible values are any decimal value greater than zero or -1 which indicates the willingness to pay any on-demand price. For more details on spot pricing, see [spot VMs pricing](https://docs.microsoft.com/azure/virtual-machines/spot-vms#pricing)"
     },
+    "ScaleDownMode": {
+      "type": "string",
+      "enum": [
+        "Delete",
+        "Deallocate"
+      ],
+      "x-ms-enum": {
+        "name": "ScaleDownMode",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Delete",
+            "description": "Delete mode wil create new instances during scale up and remove instances during scale down."
+          },
+          {
+            "value": "Deallocate",
+            "description": "Deallocate mode will attempt to start deallocated instances (if they exist) during scale up and deallocate instances during scale down."
```

```suggestion
            "description": "Attempt to start deallocated instances (if they exist) during scale up and deallocate instances during scale down."
```

Is there an article somewhere explaining what the difference is between delete and deallocate? We should probably link to it specifically in this deallocate case so that we preempt the "but what actually is a deallcoate? Am I still changed? What happens to data on the VMs if any, etc"

---

Not officially - the closest I have found was this:
https://docs.microsoft.com/en-us/azure/virtual-machines/states-billing

---

for deallocate, you aren't charged for compute. You are charged for disk (and maybe network)

---

That article seems ok to me -- might be worth linking it. I believe you can use the markdown style []() for linking and if you do, make sure to remove the `en-us` from the URL when you put it into the Swagger (so that they can get redirected to the correct one for their language as appropriate).